pkg/controller/pxc/controller.go

```diff
@@ -338,6 +345,76 @@ func (r *ReconcilePerconaXtraDBCluster) deploy(cr *api.PerconaXtraDBCluster) err
 	return nil
 }
 
+func createSSL(cr *api.PerconaXtraDBCluster, index string) ([]byte, []byte, []byte, error) {
```

Comment:
I'd like to see either named return parameters or (as for me even better) return some `struct` with certs and keys instead of `[]byte, []byte, []byte`. Otherwise, it's hard to understand for function user what all these []bytes does mean.